tasks/main.yml

```diff
@@ -20,6 +20,14 @@
         - "atom-deps"
       when:
         - ansible_os_family == "RedHat"
+        - atom_php_version|int < 74
+
+    - include: "deps-rh-74.yml"
```

Comment:
(this also nitpicky) Would it be better to use a file name like "deps-php-rh-74.yml" to make it clear it is about php 7.4 (and not about RedHat 7.4) ?